packages/server/lib/util/terminal.js

```diff
@@ -39,30 +25,26 @@ const getBordersLength = (left, right) => {
 }
 
 const convertDecimalsToNumber = function (colWidths, cols) {
-  let diff
-  const sum = _.sum(colWidths)
+  // const sum = _.sum(colWidths)
```

Comment:
a few commented out lines

Reply:
Thanks for pointing this out. I removed this while experimenting - need to add back.

Reply:
Actually, I might just remove this 😬